---
title: "MADA Course - Pre-view of the Data Analysis Project"
project: "Comparison of *Trypanosoma cruzi* parasite composition across hosts and time points"
author: "Aylla von Ermland"
format:
  html:
    toc: false
    number-sections: true
    highlight-style: github
  docx: default
---

```{r, echo=FALSE, message=FALSE}
# load a few R packages
library(here)
library(knitr)
```

**Authors**  

* Aylla von Ermland$^{1,2,*}$ 

**Author affiliations**  

1. Department of Cellular Biology, University of Georgia, Athens, GA, USA.
2. Center for Tropical and Emerging Global Diseases, Athens, GA, USA.


$\land$ Corresponding author: ayllae@uga.edu


{{< pagebreak >}}


# Introduction 

# General Background Information

I study the protozoan parasite Trypanosoma cruzi that causes Chagas disease. This disease is of interest because understanding parasite infection dynamics is important for improving treatment regimens and understanding how the disease affects humans. The goal of this project is to investigate if reinfection occurs among natural infected hosts in a relative small window of time (even when the hosts have been previously treated for the disease).

The parasites analyzed in this study were isolated from naturally infected hosts, including dogs and macaques, in two different locations in Texas. Dogs were sampled from ranches, while macaques were housed in non-human primate research facilities with an indoor/outdoor configuration. We received samples from these locations in collaboration of Texas A&M University. Because both the *T. cruzi* parasite and its vector (kissing bugs) are endemic among wild animals in Texas, these animals are naturally exposed and become infected frequently. All animals included in this study were previously treated with the standard drug regimen for Chagas disease (Benznidazole). A few months (or years) after treatment, these animals had a relapse in parasitemia and became sick again, but it is unclear whether this is due to treatment failure or reinfection over time, given continued exposure to the parasite and vector.

# Description of data and data source

The data consist of a table containing frequencies of amplicon sequence clusters (or variants) from 16 samples of parasite DNA obtained through amplicon sequencing (nanopore). There are five parasite samples from dogs and eleven samples from macaques, collected across multiple time points over several months for each animal.

PCR amplification targeted the Hyper Variable Region (HVR) of the genome (200â€“400 bp), which is located in an organelle called kinetoplast DNA (kDNA). This region consists of multiple copies within the genome. Sequenced HVR reads were classified into clusters using k-mer analysis through a custom Python pipeline. The resulting data are stored in the file cluster_relative_frequencies_per_sample.csv, which contains the relative frequencies of clusters for each sample.

```{r}
library(readr)
library(dplyr)
library(here)

# Define the object by assigning the result to a variable
cluster_data <- read_csv(here("Individual-project", "cluster_relative_frequencies_per_sample.csv"))

# Inspect the data
glimpse(cluster_data)
head(cluster_data)
summary(cluster_data)
```

# Questions/Hypotheses to be addressed

The main hypothesis addressed in this project is whether natural parasite reinfection occurs in the same host over time. In order to answer this question, I will compare the HVR sequene profile of parasites collected from host blood at different time points to determine whether parasite composition remains the same or differs across samples. If the cluster frequency profiles are the same/correlated, this suggests the parasite came from the same initial infection and if they present differences, may indicate reinfection. 

Additional questions can be explored using this dataset such as whether parasite composition differs between hosts, such as dogs versus macaques, suggesting host-related differences in parasite composition. Another question is whether samples collected in the same month are more closely related to each other, because of the kissing bug season. Overall, the correlation and relatedness among samples are examined to assess patterns associated with host type, time point, and sampling period.

I expect to observe reinfection in at least 10% of the samples, based on treatment data showing that approximately 10% of treated animals experience an increase in parasitemia after some time.

# Methods - Data analysis approach 

Initial exploratory analyses include plotting frequency bar plots for cluster composition across samples and to acess overall patterns. A correlation matrix will then be generated based on shared clusters between samples. This will allow testing whether samples from the same host at different time points are more closely related to each other than to samples from other hosts. Higher correlation with samples from other animals may indicate reinfection.

Additional analyses include evaluating whether host type (dogs or macaques) is a predictive variable for parasite composition and testing whether samples collected during the same month are correlated across hosts. These analyses will help assess temporal, host-related, and potential geographic patterns in parasite composition.

