<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Aylla von Ermland">

<title>Comparison of Trypanosoma cruzi parasite composition across hosts and time points – Aylla Ermland Data Analysis Portfolio</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-234273d1456647dabc34a594ac50e507.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-45a984b55c978f8c4117b5b9aec951d9.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Aylla Ermland Data Analysis Portfolio</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../aboutme.html"> 
<span class="menu-text">About Me</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-projects" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">Projects</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-projects">    
        <li>
    <a class="dropdown-item" href="../starter-analysis-exercise/starter-analysis-exercise.html">
 <span class="dropdown-text">Starter Analysis Exercise</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../ready-exercise/ready-exercise.html">
 <span class="dropdown-text">READy Exercise</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../Individual-project/Manuscript_copy.html">
 <span class="dropdown-text">Individual Project</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../data-exercise/data-exercise.html">
 <span class="dropdown-text">Data Exercise</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/ayllae/AyllaErmland-portfolio"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    <div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="Manuscript_copy.docx"><i class="bi bi-file-word"></i>MS Word</a></li></ul></div></div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Comparison of <em>Trypanosoma cruzi</em> parasite composition across hosts and time points</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Aylla von Ermland </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<div class="cell">
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'knitr' was built under R version 4.4.3</code></pre>
</div>
</div>
<p><strong>Authors</strong><br>
* Aylla von Ermland<span class="math inline">\(^{1,2,*}\)</span></p>
<ol type="1">
<li><p>Department of Cellular Biology, University of Georgia, Athens, GA, USA.</p></li>
<li><p>Center for Tropical and Emerging Global Diseases, Athens, GA, USA.</p></li>
</ol>
<p>Corresponding author: ayllae@uga.edu</p>
<div style="page-break-after: always;"></div>
<section id="introduction" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Introduction</h1>
</section>
<section id="general-background-information" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> General Background Information</h1>
<p>I study the protozoan parasite <em>Trypanosoma cruzi</em> that causes Chagas disease. This disease is of interest because understanding parasite infection dynamics is important for improving treatment regimens and understanding how the disease affects humans. Our previous data shows that when naturally infected dogs are once drug-treated, the disease can relapse months or years after, but it is still unkown if this occurs because of treatment failure (which is most of the times assumed) or if it is because of a reinfection, which in this case a new treatment would be required. The goal of this project is to investigate if reinfection occurs among natural infected hosts in a relative small window of time. I have just received this data and have not analyzed it, so the MADA individual project be a great opportunity.</p>
<p>The parasites analyzed in this study were isolated from blood of naturally infected hosts, including dogs and macaques, in two different locations in Texas, USA. Dogs were sampled from ranches, while macaques were housed in non-human primate research facilities with an indoor/outdoor configuration. We received samples from these locations in collaboration of Texas A&amp;M University. Because both the <em>T. cruzi</em> parasite and its vector (kissing bugs) are endemic among wild animals in Texas, these animals are naturally exposed and become infected frequently. All animals included in this study were previously treated with the standard drug regimen for Chagas disease (Benznidazole). A few months (or years) after treatment, these animals had a relapse in parasitemia and became sick again, but it is unclear whether this is due to treatment failure or reinfection over time given continued exposure to the parasite and vector.</p>
</section>
<section id="description-of-data-and-data-source" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Description of data and data source</h1>
<p>The data consist of a table containing frequencies of amplicon sequence clusters (or variants) from 16 samples of parasite DNA obtained through amplicon sequencing (nanopore). There are five parasite samples from dogs and eleven samples from macaques, collected across multiple time points over several months for each animal (see below):</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Sample_info.png" class="img-fluid figure-img"></p>
<figcaption>Sample information</figcaption>
</figure>
</div>
<p>Using blood DNA samples, I performed a PCR amplification of the Hyper Variable Region (HVR) of the genome (200–400 bp), which is located in an organelle called kinetoplast DNA (kDNA). This region consists of multiple copies within the genome. Sequenced HVR reads were classified into clusters using k-mer analysis through a custom Python pipeline. Each cluster represents a group or variant of this HVR. The resulting data are stored in the file cluster_relative_frequencies_per_sample.csv, which contains the relative frequencies of clusters for each sample.</p>
<p>The csv file below represents each sample information, where each row is a sample, columns are named animal_id; month; year; animal_spp; and the Cluster 1; Cluster 2; ….Each value represents the relative frequency of that sequence cluster in that particular sample.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: package 'readr' was built under R version 4.4.3</code></pre>
</div>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'dplyr'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:stats':

    filter, lag</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union</code></pre>
</div>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>here() starts at /Users/ayllaermland/Downloads/BIOS8060E/AyllaErmland-portfolio</code></pre>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Define the object by assigning the result to a variable</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>cluster_data <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(<span class="fu">here</span>(<span class="st">"Individual-project"</span>, <span class="st">"cluster_relative_frequencies_per_sample2.csv"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 18 Columns: 42</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr  (3): animal_id, month, animal_spp
dbl (39): year, cluster 1, cluster 2, cluster 3, cluster 4, cluster 5, clust...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Inspect the data</span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="fu">glimpse</span>(cluster_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Rows: 18
Columns: 42
$ animal_id    &lt;chr&gt; NA, "14159", "14159", "15039", "15039", "15039", "15056",…
$ month        &lt;chr&gt; NA, "May", "Oct", "June", "May", "Nov", "June", "Nov", "F…
$ year         &lt;dbl&gt; NA, 2022, 2022, 2022, 2022, 2022, 2022, 2022, 2024, 2025,…
$ animal_spp   &lt;chr&gt; NA, "macaque", "macaque", "macaque", "macaque", "macaque"…
$ `cluster 1`  &lt;dbl&gt; 1.00000000, 0.13548490, 0.08463779, 0.10771841, 0.1207078…
$ `cluster 2`  &lt;dbl&gt; 10.000000000, 0.035788465, 0.037313433, 0.098388465, 0.05…
$ `cluster 3`  &lt;dbl&gt; 11.000000000, 0.147946956, 0.290134692, 0.002544529, 0.00…
$ `cluster 4`  &lt;dbl&gt; 12.000000000, 0.029397667, 0.033127048, 0.055979644, 0.04…
$ `cluster 5`  &lt;dbl&gt; 13.000000000, 0.000319540, 0.001638151, 0.000000000, 0.00…
$ `cluster 6`  &lt;dbl&gt; 14.000000000, 0.107365394, 0.215689843, 0.002261804, 0.00…
$ `cluster 7`  &lt;dbl&gt; 15.000000000, 0.007988497, 0.000000000, 0.000848176, 0.00…
$ `cluster 8`  &lt;dbl&gt; 16.000000000, 0.001278159, 0.000546050, 0.000000000, 0.00…
$ `cluster 9`  &lt;dbl&gt; 17.000000000, 0.000639080, 0.000728067, 0.000282725, 0.00…
$ `cluster 10` &lt;dbl&gt; 18.000000000, 0.000000000, 0.000182017, 0.040712468, 0.22…
$ `cluster 11` &lt;dbl&gt; 19.000000000, 0.000798850, 0.000910084, 0.000000000, 0.00…
$ `cluster 12` &lt;dbl&gt; 2.000000000, 0.012621825, 0.014743356, 0.026293469, 0.047…
$ `cluster 13` &lt;dbl&gt; 20.000000000, 0.000159770, 0.000728067, 0.000000000, 0.00…
$ `cluster 14` &lt;dbl&gt; 21.000000000, 0.000000000, 0.000000000, 0.000000000, 0.00…
$ `cluster 15` &lt;dbl&gt; 22.000000000, 0.000000000, 0.000000000, 0.000000000, 0.00…
$ `cluster 16` &lt;dbl&gt; 23.000000000, 0.009745966, 0.010010921, 0.015549901, 0.00…
$ `cluster 17` &lt;dbl&gt; 24.000000000, 0.004952868, 0.000000000, 0.000000000, 0.00…
$ `cluster 18` &lt;dbl&gt; 25.000000000, 0.000639080, 0.000910084, 0.000000000, 0.00…
$ `cluster 19` &lt;dbl&gt; 26.000000000, 0.000159770, 0.000546050, 0.000000000, 0.00…
$ `cluster 20` &lt;dbl&gt; 27.000000000, 0.000000000, 0.000182017, 0.024879842, 0.14…
$ `cluster 21` &lt;dbl&gt; 28.000000000, 0.001437929, 0.000000000, 0.000848176, 0.00…
$ `cluster 22` &lt;dbl&gt; 29.000000000, 0.100814827, 0.023844194, 0.000282725, 0.00…
$ `cluster 23` &lt;dbl&gt; 3.00000000, 0.13037226, 0.07426283, 0.14645180, 0.0954286…
$ `cluster 24` &lt;dbl&gt; 30.000000000, 0.000958620, 0.000364033, 0.070681368, 0.00…
$ `cluster 25` &lt;dbl&gt; 31.000000000, 0.081482665, 0.015835457, 0.000000000, 0.00…
$ `cluster 26` &lt;dbl&gt; 32.000000000, 0.000319540, 0.000546050, 0.000000000, 0.00…
$ `cluster 27` &lt;dbl&gt; 33.000000000, 0.000159770, 0.000182017, 0.000000000, 0.00…
$ `cluster 28` &lt;dbl&gt; 34.000000000, 0.009745966, 0.018019658, 0.000000000, 0.00…
$ `cluster 29` &lt;dbl&gt; 35.000000000, 0.000000000, 0.000000000, 0.000000000, 0.00…
$ `cluster 30` &lt;dbl&gt; 36.000000000, 0.000000000, 0.000000000, 0.000000000, 0.00…
$ `cluster 31` &lt;dbl&gt; 37.000000000, 0.000000000, 0.000000000, 0.000000000, 0.00…
$ `cluster 32` &lt;dbl&gt; 38.000000000, 0.000000000, 0.000182017, 0.000000000, 0.00…
$ `cluster 33` &lt;dbl&gt; 4.00000000, 0.08691484, 0.05496906, 0.04721515, 0.0644617…
$ `cluster 34` &lt;dbl&gt; 5.000000000, 0.001917239, 0.002184201, 0.000282725, 0.000…
$ `cluster 35` &lt;dbl&gt; 6.000000000, 0.047771209, 0.056971241, 0.187164264, 0.097…
$ `cluster 36` &lt;dbl&gt; 7.000000000, 0.040262023, 0.055515107, 0.171048912, 0.090…
$ `cluster 37` &lt;dbl&gt; 8.000000000, 0.001278159, 0.003822352, 0.000000000, 0.000…
$ `cluster 38` &lt;dbl&gt; 9.000000000, 0.001278159, 0.001274117, 0.000565451, 0.000…</code></pre>
</div>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(cluster_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 6 × 42
  animal_id month  year animal_spp `cluster 1` `cluster 2` `cluster 3`
  &lt;chr&gt;     &lt;chr&gt; &lt;dbl&gt; &lt;chr&gt;            &lt;dbl&gt;       &lt;dbl&gt;       &lt;dbl&gt;
1 &lt;NA&gt;      &lt;NA&gt;     NA &lt;NA&gt;            1          10         11      
2 14159     May    2022 macaque         0.135       0.0358     0.148  
3 14159     Oct    2022 macaque         0.0846      0.0373     0.290  
4 15039     June   2022 macaque         0.108       0.0984     0.00254
5 15039     May    2022 macaque         0.121       0.0590     0.00232
6 15039     Nov    2022 macaque         0.209       0.0641     0.0109 
# ℹ 35 more variables: `cluster 4` &lt;dbl&gt;, `cluster 5` &lt;dbl&gt;, `cluster 6` &lt;dbl&gt;,
#   `cluster 7` &lt;dbl&gt;, `cluster 8` &lt;dbl&gt;, `cluster 9` &lt;dbl&gt;,
#   `cluster 10` &lt;dbl&gt;, `cluster 11` &lt;dbl&gt;, `cluster 12` &lt;dbl&gt;,
#   `cluster 13` &lt;dbl&gt;, `cluster 14` &lt;dbl&gt;, `cluster 15` &lt;dbl&gt;,
#   `cluster 16` &lt;dbl&gt;, `cluster 17` &lt;dbl&gt;, `cluster 18` &lt;dbl&gt;,
#   `cluster 19` &lt;dbl&gt;, `cluster 20` &lt;dbl&gt;, `cluster 21` &lt;dbl&gt;,
#   `cluster 22` &lt;dbl&gt;, `cluster 23` &lt;dbl&gt;, `cluster 24` &lt;dbl&gt;, …</code></pre>
</div>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(cluster_data)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>  animal_id            month                year       animal_spp       
 Length:18          Length:18          Min.   :2022   Length:18         
 Class :character   Class :character   1st Qu.:2022   Class :character  
 Mode  :character   Mode  :character   Median :2022   Mode  :character  
                                       Mean   :2023                     
                                       3rd Qu.:2023                     
                                       Max.   :2025                     
                                       NA's   :2                        
   cluster 1         cluster 2           cluster 3           cluster 4        
 Min.   :0.01861   Min.   : 0.003515   Min.   : 0.000000   Min.   : 0.003775  
 1st Qu.:0.05777   1st Qu.: 0.006797   1st Qu.: 0.000572   1st Qu.: 0.022444  
 Median :0.13548   Median : 0.032615   Median : 0.002317   Median : 0.030368  
 Mean   :0.17213   Mean   : 0.620701   Mean   : 0.674816   Mean   : 0.734027  
 3rd Qu.:0.16402   3rd Qu.: 0.060439   3rd Qu.: 0.004583   3rd Qu.: 0.041373  
 Max.   :1.00000   Max.   :10.000000   Max.   :11.000000   Max.   :12.000000  
 NA's   :1         NA's   :1           NA's   :1           NA's   :1          
   cluster 5           cluster 6           cluster 7        
 Min.   : 0.000000   Min.   : 0.000000   Min.   : 0.000000  
 1st Qu.: 0.000320   1st Qu.: 0.000196   1st Qu.: 0.000211  
 Median : 0.003051   Median : 0.001290   Median : 0.002761  
 Mean   : 0.787982   Mean   : 0.843473   Mean   : 0.898652  
 3rd Qu.: 0.057548   3rd Qu.: 0.002317   3rd Qu.: 0.007988  
 Max.   :13.000000   Max.   :14.000000   Max.   :15.000000  
 NA's   :1           NA's   :1           NA's   :1          
   cluster 8           cluster 9           cluster 10       
 Min.   : 0.000000   Min.   : 0.000000   Min.   : 0.000000  
 1st Qu.: 0.000546   1st Qu.: 0.000283   1st Qu.: 0.000000  
 Median : 0.009303   Median : 0.003622   Median : 0.000182  
 Mean   : 0.958204   Mean   : 1.017494   Mean   : 1.077885  
 3rd Qu.: 0.036254   3rd Qu.: 0.043678   3rd Qu.: 0.000425  
 Max.   :16.000000   Max.   :17.000000   Max.   :18.000000  
 NA's   :1           NA's   :1           NA's   :1          
   cluster 11          cluster 12        cluster 13          cluster 14       
 Min.   : 0.000000   Min.   :0.00674   Min.   : 0.000000   Min.   : 0.000000  
 1st Qu.: 0.000799   1st Qu.:0.01262   1st Qu.: 0.000587   1st Qu.: 0.000000  
 Median : 0.005789   Median :0.02629   Median : 0.002860   Median : 0.000000  
 Mean   : 1.131724   Mean   :0.23042   Mean   : 1.190034   Mean   : 1.251486  
 3rd Qu.: 0.029114   3rd Qu.:0.18479   3rd Qu.: 0.031843   3rd Qu.: 0.001176  
 Max.   :19.000000   Max.   :2.00000   Max.   :20.000000   Max.   :21.000000  
 NA's   :1           NA's   :1         NA's   :1           NA's   :1          
   cluster 15          cluster 16          cluster 17       
 Min.   : 0.000000   Min.   : 0.000827   Min.   : 0.000000  
 1st Qu.: 0.000000   1st Qu.: 0.004036   1st Qu.: 0.000000  
 Median : 0.000179   Median : 0.010011   Median : 0.000425  
 Mean   : 1.309116   Mean   : 1.364255   Mean   : 1.421105  
 3rd Qu.: 0.000632   3rd Qu.: 0.019875   3rd Qu.: 0.004953  
 Max.   :22.000000   Max.   :23.000000   Max.   :24.000000  
 NA's   :1           NA's   :1           NA's   :1          
   cluster 18          cluster 19          cluster 20         cluster 21       
 Min.   : 0.000000   Min.   : 0.000000   Min.   :0.00e+00   Min.   : 0.000000  
 1st Qu.: 0.000639   1st Qu.: 0.000211   1st Qu.:0.00e+00   1st Qu.: 0.001438  
 Median : 0.010337   Median : 0.002097   Median :0.00e+00   Median : 0.005179  
 Mean   : 1.481129   Mean   : 1.539841   Mean   :1.60e+00   Mean   : 1.654248  
 3rd Qu.: 0.014559   3rd Qu.: 0.026089   3rd Qu.:4.25e-04   3rd Qu.: 0.006661  
 Max.   :25.000000   Max.   :26.000000   Max.   :2.70e+01   Max.   :28.000000  
 NA's   :1           NA's   :1           NA's   :1          NA's   :1          
   cluster 22          cluster 23        cluster 24          cluster 25       
 Min.   : 0.000000   Min.   :0.02522   Min.   : 0.000000   Min.   : 0.000000  
 1st Qu.: 0.000000   1st Qu.:0.05714   1st Qu.: 0.000000   1st Qu.: 0.000000  
 Median : 0.000000   Median :0.10853   Median : 0.000196   Median : 0.000000  
 Mean   : 1.713274   Mean   :0.27664   Mean   : 1.774044   Mean   : 1.829294  
 3rd Qu.: 0.000283   3rd Qu.:0.14645   3rd Qu.: 0.000421   3rd Qu.: 0.000223  
 Max.   :29.000000   Max.   :3.00000   Max.   :30.000000   Max.   :31.000000  
 NA's   :1           NA's   :1         NA's   :1           NA's   :1          
   cluster 26         cluster 27         cluster 28         cluster 29      
 Min.   : 0.00000   Min.   : 0.00000   Min.   : 0.00000   Min.   : 0.00000  
 1st Qu.: 0.00032   1st Qu.: 0.00018   1st Qu.: 0.00000   1st Qu.: 0.00000  
 Median : 0.00434   Median : 0.00165   Median : 0.00000   Median : 0.00000  
 Mean   : 1.88837   Mean   : 1.94300   Mean   : 2.00167   Mean   : 2.05986  
 3rd Qu.: 0.01335   3rd Qu.: 0.00345   3rd Qu.: 0.00025   3rd Qu.: 0.00216  
 Max.   :32.00000   Max.   :33.00000   Max.   :34.00000   Max.   :35.00000  
 NA's   :1          NA's   :1          NA's   :1          NA's   :1         
   cluster 30         cluster 31         cluster 32         cluster 33     
 Min.   : 0.00000   Min.   : 0.00000   Min.   : 0.00000   Min.   :0.01964  
 1st Qu.: 0.00000   1st Qu.: 0.00000   1st Qu.: 0.00000   1st Qu.:0.05497  
 Median : 0.00058   Median : 0.00000   Median : 0.00018   Median :0.06446  
 Mean   : 2.11825   Mean   : 2.17706   Mean   : 2.23572   Mean   :0.29411  
 3rd Qu.: 0.00126   3rd Qu.: 0.00121   3rd Qu.: 0.00055   3rd Qu.:0.07926  
 Max.   :36.00000   Max.   :37.00000   Max.   :38.00000   Max.   :4.00000  
 NA's   :1          NA's   :1          NA's   :1          NA's   :1        
   cluster 34         cluster 35         cluster 36         cluster 37      
 Min.   :0.000000   Min.   :0.005375   Min.   :0.003453   Min.   :0.000000  
 1st Qu.:0.001917   1st Qu.:0.009558   1st Qu.:0.006797   1st Qu.:0.001278  
 Median :0.041051   Median :0.047771   Median :0.040262   Median :0.035680  
 Mean   :0.334348   Mean   :0.402256   Mean   :0.456781   Mean   :0.504833  
 3rd Qu.:0.075828   3rd Qu.:0.091702   3rd Qu.:0.085971   3rd Qu.:0.064013  
 Max.   :5.000000   Max.   :6.000000   Max.   :7.000000   Max.   :8.000000  
 NA's   :1          NA's   :1          NA's   :1          NA's   :1         
   cluster 38      
 Min.   :0.000000  
 1st Qu.:0.001274  
 Median :0.006482  
 Mean   :0.561185  
 3rd Qu.:0.073087  
 Max.   :9.000000  
 NA's   :1         </code></pre>
</div>
</div>
</section>
<section id="hypotheses-to-be-addressed" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Hypotheses to be addressed</h1>
<p>The main hypothesis addressed in this project is whether natural parasite reinfection occurs in the same host over time. In order to answer this question, I will compare the HVR cluster profile of parasites collected from host blood at different time points to determine whether parasite composition remains the same or differs across samples. If the cluster frequency profiles are the same/correlated, this suggests the parasite came from the same initial infection and if they present differences, may indicate reinfection.</p>
<p>Additional questions can be explored using this dataset such as whether parasite composition differs between hosts, such as dogs versus macaques, suggesting host-related differences in parasite composition. Another question is whether samples collected in the same month are more closely related to each other, because of the kissing bug season. Overall, the correlation and relatedness among samples are examined to assess patterns associated with host type, time point, and sampling period.</p>
<p>I expect to observe reinfection in at least 10% of the samples, based on treatment data showing that approximately 10% of treated animals experience a later relapse of the disease.</p>
</section>
<section id="methods-data-analysis-approach" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Methods: Data analysis approach</h1>
<p>Initial exploratory analyses include plotting frequency bar plots for cluster composition across samples and to acess overall patterns. A correlation matrix will then be generated based on shared clusters between samples. This will allow testing whether samples from the same host at different time points are more closely related to each other than to samples from other hosts. Higher correlation with samples from other animals may indicate reinfection.</p>
<p>Additional analyses include evaluating whether host type (dogs or macaques) is a predictive variable for parasite composition and testing whether samples collected during the same month are correlated across hosts. These analyses will help assess temporal, host-related, and potential geographic patterns in parasite composition.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>